praw/models/reddit/subreddit.py

```diff
@@ -1586,8 +1586,13 @@ def conversations(self, after=None, limit=25, other_subreddits=None,
 
         response = self.subreddit._reddit.get(
             API_PATH['modmail_conversations'], params=params)
-        return [self(conversation_id)
-                for conversation_id in response['conversationIds']]
+        conversations = []
+        for conversation_id in response['conversationIds']:
```

Comment:
Is the benefit of this approach that the data is already available thus a subsequent request isn't required?

If so, perhaps the test case can be updated to make an assertion about one of those attributes. And given that the cassette is already created, it should not result in a new request -- whereas the code in the parent commit would have.